### ISSUE TITLE
Server to Client Meta Data not work

### ISSUE THREAD

**Issue by yacthMon** [sffc/socketio-file-upload#82]
Hi I have using this library for my socket server and I have issue about Server to Client meta data.
It's not work in my case even I follow the sample code 

Here is my code
``` javascript 
// Server
  uploader.on("saved", function (event) {
    event.file.clientDetail.hello = "world";
  });
```
``` javascript 
// Client
  this.uploader.addEventListener("complete", function (event) {
    console.log(`Complete upload : ${event.detail.hello}`)
    console.log(event)
  });
```
And `event` in client after `complete` got empty detail.

![image](https://user-images.githubusercontent.com/9114148/46401822-48468300-c728-11e8-9a97-6da7db53adaf.png)

But Client to Server Meta Data work correclty.

Am I doing anything wrong ?

Thank you :)

**Comment by yacthMon** [comment]
This [commit](https://github.com/sffc/socketio-file-upload/commit/85695ff4b308dbefe590832f16533f0df03a2e28) change emit saved event after write stream make Server to Client Meta Data not work for event `saved` anymore.

So to do Server to Client Meta data you need to use another event like `complete` instead.

**Comment by sffc** [comment]
Thank you for the report!  I fixed the issue, added a test, and published v0.6.3.
